corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -234,10 +234,14 @@ hqDefine('app_manager/js/app_manager', function () {
     var _initMenuItemSorting = function () {
         nestChildModules();
         initDragHandles();
-        $('.sortable').each(function () {
-            initSortable($(this));
-        });
-        $('.sort-action').hide();
+        if (modulesWereReordered()) {
+            promptToSaveOrdering();
+        } else {
+            $('.sortable').each(function () {
+                initSortable($(this));
+            });
+            $('.sort-action').hide();
```

Comment:
Nitpick: It's a good habit to use `addClass('hide')` instead of `.hide()`, because the latter directly updates an inline style, which can screw with other styling.

Reply:
Actually, do you know of any reason that's not just hidden in the html?  Why bother doing that in JS?

Reply:
Hmm, I don't know. I'd definitely favor doing it in html.